docs/adguard-for-android/solving-problems/low-level-settings.md

```diff
@@ -108,21 +108,25 @@ Every encrypted Internet connection has an unencrypted part. This is the very fi
 
 #### OCSP checking
 
-Once enabled, this option runs asynchronous OCSP checks to check whether the website’s SSL certificate is revoked.
+This option runs asynchronous OCSP checks to check whether the website’s SSL certificate is revoked.
 
-If the OCSP check is completed within the minimum timeout, AdGuard will immediately block the connection if the certificate is revoked or establish the connection if the certificate is valid.
+If the OCSP check is completed within the minimum timeout, AdGuard immediately blocks the connection if the certificate is revoked or establishes the connection if the certificate is valid.
 
-If the verification takes too long, AdGuard will establish a connection and continue checking the certificate in the background. If it is revoked, current and future connections to the domain will be blocked.
+If the verification takes too long, AdGuard establishes a connection and continues checking the certificate in the background. If it is revoked, current and future connections to the domain are blocked.
 
 #### Redirect DNS-over-HTTPS requests
 
-IF enabled, DNS-over-HTTPS requests will be redirected to the DNS Protection module. We recommend disabling fallback upstreams and use only encrypted DNS servers to maintain privacy.
+This option makes AdGuard redirect DNS-over-HTTPS requests to the local DNS proxy in addition to plain DNS requests. We recommend disabling fallback upstreams and using only encrypted DNS servers to maintain privacy.
 
-### Outbound Proxy
+#### Filter HTTP/3
+
+This option makes AdGuard filter requests sent over HTTP/3 in addition to other request types.
+
+### Outbound proxy
 
 #### Show the Filter DNS requests setting
 
-When this feature is enabled, the string *Filter DNS requests* appears in the *Settings ➝ Filtering ➝ Network ➝ Proxy ➝ Proxy server ➝ Add proxy server* section with the switch next to it. By toggling the switch, you can enable filtering of DNS requests passing through the proxy.
+This option displays the *Filter DNS requests* switch in the *Add proxy server* dialog. By toggling the switch, you can enable filtering of DNS requests that pass through the specified proxy.
```

Comment:
If this is enabled, the *Filter DNS requests* switch will be displayed in the *Add proxy server* dialog. Use it to enable filtering of DNS requests passing through the specified proxy.